### ISSUE TITLE
I'm trying Slicing concept but not sure where can I check that logical slice.

### ISSUE THREAD

**Issue by PallaviTharigonda** [aligungr/UERANSIM#332]
Hi @aligungr , I'm trying to do slicing on top of our physical network . So in open5gs core network I've changed sst:1 to sst: 2 (for URLLC) and configured in UERANSIM ,open5gs-gnb.yaml and open5gs-ue.yaml .In UI also I've given sst: 2 for each imsi Id. But my doubt is where can I check this logical part?

**Comment by aligungr** [comment]
Hi @PallaviTharigonda

I'm not really sure if we can currently "test" the concept of network slicing in either UERANSIM or Open5GS side.

Regards

**Comment by PallaviTharigonda** [comment]
@aligungr So you are telling we can't implement slicing or we can't check?

**Comment by aligungr** [comment]
@PallaviTharigonda I mean I'm not sure about a testing (or checking) procedure for network slicing. We already implemented slicing in the UE side

**Comment by aligungr** [comment]
@PallaviTharigonda I mean I'm not sure about a testing (or checking) procedure for network slicing. We already implemented slicing in the UE side

**Comment by PallaviTharigonda** [comment]
@aligungr ya I saw that ,1 default slice is already available .So I'm asking if I wanted to change sst: 1 to sst: 2 is that means 2 slices are available now?

**Comment by aligungr** [comment]
@PallaviTharigonda Yes, you can configure UE to use any network slice. But the main configurations and flows are on the network side. UE only indicates that I want to use this specific slices, for example.
